deploy/helm/sumologic/values.yaml

```diff
@@ -106,7 +106,7 @@ sumologic:
     # How many spans per request should be send to receiver
     spans_per_request: 100
 
-  ## Format to post logs into Sumo. json, json_merge, or text
+  ## Format to post logs into Sumo. json, json_merge, or text	fluentd:
```

Comment:
This looks weird, might be some sort of copy and paste mistake on my part. Can we remove `fluentd:` from the line?

Reply:
Done :)